integration/combination/test_api_with_gateway_responses.py

```diff
@@ -1,10 +1,14 @@
+import logging
 from unittest.case import skipIf
 
+from tenacity import stop_after_attempt, retry_if_exception_type, after_log, wait_exponential, retry, wait_random
+
 from integration.helpers.base_test import BaseTest
-from integration.helpers.deployer.utils.retry import retry
 from integration.helpers.resource import current_region_does_not_support
 from integration.config.service_names import GATEWAY_RESPONSES
 
+LOG = logging.getLogger(__name__)
```

Comment:
It seems that we added this logger but I can't see any logging output, is this redundant?

Reply:
It is used in line 44.

Reply:
Oh I see, library uses it